src/main/resources/application-local.yml

```diff
@@ -0,0 +1,31 @@
+#
+# The "-local" profile is for running the API service locally against pre-existing
+# docker containers that are listening on a known port numbers for a postgres database (5432).
+#
+
+# Most values are inherited from - application.yml - except the following:
+spring:
+  datasource:
+    url: 'jdbc:postgresql://${DB_SERVER}/${DB_NAME}?sslmode=prefer'
```

Comment:
We could possibly just hard code the values for the db and users here to save specifying them given they rarely change when running locally.

Reply:
They're overridden in docker-compose and run-local scripts, so best left with env vars, I think.